liquidator/liquidator.js

```diff
@@ -125,14 +125,12 @@ class Liquidator {
     });
 
     // If an override is provided, use that price. Else, get the latest price from the price feed.
-    const price = liquidatorOverridePrice ? this.toBN(liquidatorOverridePrice) : this.priceFeed.getCurrentPrice();
+    const price = liquidatorOverridePrice
+      ? this.toBN(liquidatorOverridePrice.toString())
+      : this.priceFeed.getCurrentPrice();
 
     if (!price) {
-      this.logger.warn({
-        at: "Liquidator",
-        message: "Cannot liquidate: price feed returned invalid value"
-      });
-      return;
+      throw new Error("Cannot liquidate: price feed returned invalid value");
```

Comment:
Also, if there is no price the bot needs to fail more loudly than a `warn`. A side effect of this is I can test the re-try logic with invalid price feeds :)

Reply:
curious to see if you guys agree with this change.

Reply:
I'm good with this, the bot can't really do anything without a price.